Kiosk/App/CardHandler.swift

```diff
@@ -1,9 +1,14 @@
 import UIKit
-import ReactiveCocoa
+import RxSwift
 
 class CardHandler: NSObject, CFTReaderDelegate {
 
-    let cardSwipedSignal = RACSubject()
+    private let _cardStatus = BehaviorSubject(value: "")
```

Comment:
Underscores are a pattern in iOS to indicate private variables. `_cardStatus` is private so only this class may change its `value`, even though we expose its observable publicly.